tests/lib/store.sh

```diff
@@ -33,7 +33,7 @@ init_fake_refreshes(){
     local refreshable_snaps=$1
     local dir=$2
 
-    fakestore -make-refreshable "$refreshable_snaps" -dir "$dir"
+    fakestore make-refreshable "$refreshable_snaps" --dir "$dir"
```

Comment:
refreshable_snaps could be comma separated, but I don't see places using that, passing more than one snap, maybe s/refreshable_snaps/refreshable_snap/  or we should  change the callers to give dir first, and then we can use "$@"

Reply:
I like the idea of using dir first and then however many snaps. I changed the code accordingly.